sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/Generated/IncidentResource.cs

```diff
@@ -162,8 +125,45 @@ public virtual Response<IncidentCommentResource> GetIncidentComment(string incid
             return GetIncidentComments().Get(incidentCommentId, cancellationToken);
         }
 
+        /// <summary> Gets a collection of RelationResources in the Incident. </summary>
+        /// <returns> An object representing collection of RelationResources and their operations over a RelationResource. </returns>
+        public virtual RelationCollection GetRelations()
+        {
+            return GetCachedClient(Client => new RelationCollection(Client, Id));
+        }
+
         /// <summary>
-        /// Gets an incident.
+        /// Gets a relation for a given incident.
+        /// Request Path: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/incidents/{incidentId}/relations/{relationName}
+        /// Operation Id: IncidentRelations_Get
+        /// </summary>
+        /// <param name="relationName"> Relation Name. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentException"> <paramref name="relationName"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="relationName"/> is null. </exception>
+        [ForwardsClientCalls]
+        public virtual async Task<Response<RelationResource>> GetRelationAsync(string relationName, CancellationToken cancellationToken = default)
```

Comment:
Please investigate why the resource name changed from `IncidentRelationResource` to `RelationResource`. We should use the same name if possible.